man/dbxincluder.txt

```diff
@@ -0,0 +1,24 @@
+NAME
```

Comment:
You can write manpages in Sphinx/ReST too. :smiley:  
I would see that as a `.rst` file as it is easier to integrate that into the main documentation.